natgeo/mortar
# [SYTO-1028] [MORTAR] Modal issues in safari when zooming in/out

https://jira.natgeo.com/browse/SYTO-1028

#### How to test
1. ``npm run gulp serve``
2. Open Safari
3. Open a modal
4. Zoom out, the modal should expand to the entire screen.
